backend/cfg/llvmize.ml

```diff
@@ -189,15 +200,21 @@ type t =
         (* Function symbols defined so far *)
     mutable referenced_symbols : String.Set.t;
         (* Global symbols referenced so far *)
-    mutable called_functions : fun_sig String.Map.t
+    mutable called_functions : fun_sig String.Map.t;
         (* Names + signatures (args, ret) of functions called so far *)
+    mutable emit_exn_intrinsic_decls : bool
+        (* Whether to emit declarations / definitions for intrinsics / wrapper
+           functions involved in generating try blocks *)
   }
 
-let create_fun_info () =
-  { ident_gen = Ident.Gen.create ();
+let create_fun_info ~fun_name ~fun_has_try =
+  { fun_name;
+    fun_has_try;
```

Comment:
```suggestion
    fun_has_trap_handler_block;
```
I think this and `emit_exn_intrinsic_decls` should be equivalent, i.e., after we have a Pushtrap if and only if we have a trap_handler block.